tests/test_codecs.py

```diff
@@ -5,12 +5,38 @@
 import unittest
 from frontera.contrib.backends.remote.codecs.json import (Encoder as JsonEncoder,
                                                           Decoder as JsonDecoder,
-                                                          _convert, _reconvert)
+                                                          _encode_recursively, _decode_recursively)
 from frontera.contrib.backends.remote.codecs.msgpack import Encoder as MsgPackEncoder, Decoder as MsgPackDecoder
 from frontera.core.models import Request, Response
 import pytest
 
 
+def _compare_dicts(dict1, dict2):
```

Comment:
you should understand that having such deep comparisons would require extra processor cycles, and in Frontera we tend to save them and avoid doing unnecessary computation if possible. so please think again if this is really needed. may be try to find an example.